tutorial/todo-app-pipelines/template/directory/directory.cue

```diff
@@ -100,25 +100,25 @@ groupList: [...directoryv1.#Group] & [
 userList: [...directoryv1.#User] & [
 		{
 		id: {{ generateApplicationUUID "AdminUser" | quote }}
-		username:    "adminPIPE"
+		username:    "todo-app-pipelinesadmin"
```

Comment:
- outside of the immediate scope I guess but the other templates use "_" while this one uses "-". Was there a specific reason? If not and if possible I think we should use "_" everywhere for consistency.
- relatedly how about having a separator between the template name and the role, to improve readability? e.g. "todo-app-pipelines_admin" (note the "_", related to the previous bullet)

Reply:
Thank you for the feedback. I have updated the usernames and the directory names. I changed the directory name because [the e2e test](https://github.com/tailor-inc/playwright-e2e/actions/runs/6529201255) retrieves template names and executes jobs based on them. To ensure naming consistency, this change was necessary.